docs/guides/tutorial.md

```diff
@@ -0,0 +1,102 @@
+# Server-side Credentials
```

Comment:
is `docs/guides/server-side-credentials-example.md` not a rewrite of this file?

Reply:
Yes.  Removing.